dataland-frontend/src/components/forms/parts/kpiSelection/AssuranceFormField.vue

```diff
@@ -39,14 +36,10 @@
       <h4 class="mt-0">Data source</h4>
       <div class="next-to-each-other">
         <div class="flex-1">
-          <UploadFormHeader
+          <SingleSelectFormField
             :label="euTaxonomyKpiNameMappings.report ?? ''"
             :description="euTaxonomyKpiInfoMappings.report ?? ''"
-            :is-required="true"
-          />
-          <FormKit
-            type="select"
-            ignore="true"
+            :required="true"
```

Comment:
manchmal isRequired, manchmal :required="true", manchmal is-required="true", funktioniert das alles gleich?
dann würde ich das einheitlich machen.